### DISCUSSION TITLE
Function memory limit not applying

### DISCUSSION THREAD

**Discussion by imikemiller** [vercel/community#2675] [Help]
### Summary

I am attempting to increase the memory limit for an API function using the following config in my vercel.json

```
{
  "functions": {
    "pages/api/inngest.*": {
      "maxDuration": 60,
      "memory": 2048
    }
  }
}
```
I am on the professional plan and the max duration is correctly set to 60s (although this maybe the default) however the memory limit is not updating from the default 1024 to the required 2048. 

### Steps to Reproduce

```{
  "functions": {
    "pages/api/inngest.*": {
      "maxDuration": 60,
      "memory": 2048
    }
  }
}```

Using the above config no change to the memory limit is shown in the function config in the vercel console logs.

**Comment by amyegan** [comment]
Hi Mike. I wasn't able to reproduce this problem with my test app. Please share your public repo or a minimal reproduction so I can get a better look at the problem 🙏
